test/functional/test_framework/psbt.py

```diff
@@ -88,6 +92,9 @@ def serialize(self):
         for k,v in self.map.items():
             if isinstance(k, int) and 0 <= k and k <= 255:
                 k = bytes([k])
+            if isinstance(v, list):
+                assert any(type(elem) is bytes for elem in v)
```

Comment:
Since this is a new check that has been added & the current use case is of a list of only bytes (in participant pubkeys), maybe we can assert this on all instead of any to keep the checking strict to start with that can be eased in the future if needed?